demo-project/demo-project-integration-test/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.1'
+
+services:
+  app:
+    image: logb/demo-project:latest
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:postgresql://db:5432/demo
+      SPRING_DATASOURCE_USERNAME: demo
+      SPRING_DATASOURCE_PASSWORD: example
+      SPRING_RABBITMQ_HOST: rabbitmq
+      SPRING_RABBITMQ_PORT: 5672
+    ports:
+      - 8080:8080
+    depends_on:
+      - db
+      - rabbitmq
+
+  db:
+    image: postgres:13.2-alpine
+    # Uncomment this to have the DB come up when you start docker / your laptop:
+    #restart: always
```

Comment:
remove these comments, integration tests should never restart

Reply:
done